src/components/views/Rules.js

```diff
@@ -0,0 +1,139 @@
+import BaseContainer from "components/ui/BaseContainer";
+import { handleError } from "helpers/api";
+import { IoHomeOutline } from "react-icons/io5";
+import { useHistory } from "react-router-dom";
+import Header from "./Header";
+import "styles/views/Rules.scss";
+
+const Rules = () => {
+  const history = useHistory();
+  const goHome = () => {
+    try {
+      history.push("/game");
```

Comment:
it's hard to notice that the rules open in a new window. 
To make sure that the user returns to his game after reading the rules, I would suggest using `window.close()` and replace the 'Home' button with a close button.
This also makes sure that a user does not get issues by opening too many tabs.